.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: php
+
+php:
+  - 5.6
+  - 7.0
+
+before_script: composer install
+
+script: composer test
```

Comment:
According to https://github.com/composer/docker/issues/27#issuecomment-337470233 command should be `composer run-script test`.